VoltAir/Engine/android/deploy/src/org/literacyapp/voltair/VoltAirActivity.java

```diff
@@ -424,7 +428,20 @@ public void studentUpdateReceiver(String availableLetters) {
         if (availableLetters != null) {
             Log.i(LOG_TAG, availableLetters);
         }
+
+        SharedPreferences settings = getSharedPreferences(VOLTAIR_PREFS, Context.MODE_PRIVATE);
+        settings.edit().putString(PREF_STUDENT_LETTERS, availableLetters).commit();
+
         onStudentUpdateReceiver(availableLetters);
     }
 
+    public String getValidLetters() {
+        String letters = "AB";
+        SharedPreferences settings = getSharedPreferences(VOLTAIR_PREFS, Context.MODE_PRIVATE);
+        if (settings != null) {
+            letters = settings.getString(PREF_STUDENT_LETTERS, "AB");
```

Comment:
@diegowald Shouldn't this be `letters = settings.getString(PREF_STUDENT_LETTERS, letters);` since the `letters` variable is already defined above with the same value?